phing/build.yml

```diff
@@ -17,6 +17,10 @@ behat:
     - ${docroot}/profiles
     - ${repo.root}/tests/behat
   tags: '~ajax&&~experimental&&~lightningextension'
+  tags-by-build-type:
```

Comment:
Looking at this with fresh eyes, it seems like we should allow for more than dynamic behat tests. We could, for instance, allow a user to just define a bash script that gets called.